invenio_rdm_migrator/load/base.py

```diff
@@ -34,6 +29,13 @@ def _cleanup(self):
         """Cleanup data after loading."""
         pass
 
+    def _validate(self):
```

Comment:
Discussed with Zach in another PR. This helps us remove ~10 identical implementations (`return True`).